revenuecat_examples/purchase_tester/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -7,10 +7,8 @@ import Foundation
 
 import app_links
 import purchases_flutter
-import purchases_ui_flutter
```

Comment:
this file is autogenerated and does not include `import purchases_ui_flutter` because `purchases_ui_flutter` does not support macOS